slamd/design_assistant/templates/campaign_design_targets.html

```diff
@@ -0,0 +1,101 @@
+{% block content %}
+    {% from 'icons.html' import chatbot_icon %}
+    <div class="col d-flex gap-3">
+        {{ chatbot_icon(30, 30, "currentColor") }}
+        <div class="chat_message_container">
+            <div class="fw-bold mt-1 mb-2">Design Assistant</div>
+            <div class="chat_message border rounded p-2 d-inline-block">
+                <div id="design_target_options_container">
+                    <div>{{ campaign_form.design_targets_field.label(class = "mb-3") }}</div>
+                    {% for option in campaign_form.design_targets_field %}
+                        <div class="design_target_option_container d-flex justify-content-between">
+                            <div>
+                                {% if campaign_form.design_targets_field.data %}
+                                    {{ option(class_="design_target_option", disabled='disabled') }}
+                                {% else %}
+                                    {{ option(class_="design_target_option") }}
+                                {% endif %}
+                                {{ option.label.text }}
+                            </div>
+                            <div class="d-flex align-items-center">
+                                {% if loop.index == 1 %}
+                                    {{ campaign_form.target_strength_field.label(class = "flex-grow-0 flex-shrink-0 mb-3") }}
+                                    <input class="design_target_value ms-2 form-control mb-3" name="{{ option.data }}" id="campaign_form-target_strength_field" type="number" step="0.01"
+                                            {% if campaign_form.target_strength_field.data is not none %}
+                                            value="{{ campaign_form.target_strength_field.data }}"
+                                            {% endif %}
+                                           disabled
+                                    />
```

Comment:
hier und in den anderen htmls: Eigentlich schließen wir die tags nicht in einer separaten Zeile; aber ich kann damit erstmal leben